CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,72 @@
+# Autocrypt Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in the Autocrypt
+community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Showing empathy towards community members and newcomers
+* Helping and mediating in cases of upsets or conflict
+
+Examples of unacceptable behavior include:
```

Comment:
Maybe this is a bit strong, especially as some of the actions below can happen with difficult, but not malicious people.

In the sense of https://holgerkrekel.net/2015/04/09/going-beyond-codes-of-conduct/, there should be ways how we can focus on common sense and empathy.

What about "Please think twice, before you do something like this, as it may hurt people:"

All about creating an athmosphere where a CoC is not inhibiting, but improving everyone's Miteinander.... does this go into the right direction, or do you have to make such strong claims on a page like this?

Reply:
do you mean the "unacceptable" behaviour list is a bit strong?  I took it from the original text (linked at the bottom of the doc).  In the end, whatever characterization, there is always interpretation e.g. if something is "trolling" or not.  happy about PRs to this one here in any case :)

Reply:
FWIW i also dislike "acceptable" as wording.  haven't found yet a better term.